### PR TITLE
DRLM integration update within new "init" stage

### PR DESCRIPTION
**Pull request by didacog**

### CODE REVIEW

File: usr/share/rear/init/default/01_set_drlm_env.sh [hunk 1]
```diff
@@ -6,8 +6,7 @@ fi
 
 PROGS=( "${PROGS[@]}" curl )
 
-drlm_import_runtime_config
+# Needed for curl with NSS support
+LIBS=( "${LIBS[@]}" /usr/lib64/libsoftokn3.so /usr/lib64/libsqlite3.so.0 /lib64/libfreeblpriv3.so )
```
Comment (schlomo) [inline usr/share/rear/init/default/01_set_drlm_env.sh]: This code will not work for all platforms. If you want to have it in the default/ part then please modify it to be more generic.

Also, on my Ubuntu 14.04 system curl needs (according to `ldd`) only the libsqlite3.so.0, not the others. Are you sure that they are loaded on demain via dl_open()?

Did you really find out through experimentation that this is required?

**Comment by didacog** [comment]
ldd on RHEL6/Centos6 is not listing these libs: 
/usr/lib64/libsoftokn3.so 
/usr/lib64/libsqlite3.so.0 
/lib64/libfreeblpriv3.so

we identified them with strace...

File: usr/share/rear/init/default/01_set_drlm_env.sh (hunk 1 above)
Comment (schlomo) [inline usr/share/rear/init/default/01_set_drlm_env.sh]: If you really must add those libraries manually, then please use a syntax like this:
```sh
        LIBS=( 
            "${LIBS[@]}"
            /usr/lib*/libsoftokn3.so* 
            /usr/lib*/libsqlite3.so* 
            /lib*/libfreeblpriv3.so*
        )
```

And please check if the libfreeblpriv3 is really always under `/lib*`.

**Comment by didacog** [comment]
in RedHat/centos libfreeblpriv3.so is under /lib* and soft link on /usr/lib*

**Comment by schlomo** [comment]
It is actually safe to list both as LIBS will be normalized later on. That
is why it also OK to list all the soft links to a library.

**Comment by didacog** [comment]
Ok, no problem I will add the /usr/lib* link too.

**Comment by didacog** [comment]
It's ok now?

**Pull request merged**